src/Symfony/Bridge/Twig/Resources/views/Form/bootstrap_3_layout.html.twig

```diff
@@ -149,3 +154,11 @@
     {% if form is not rootform %}</span>{% else %}</div>{% endif %}
     {%- endif %}
 {%- endblock form_errors %}
+
+{# Help #}
+
+{% block form_help -%}
+    {% if help is not empty %}
```

Comment:
dont we miss whitespace control here (`-`?)
same on endif, and in other files?

Reply:
You're right, I fix it. Thanks